src/modm/driver/storage/block_device_spistack_flash.hpp

```diff
@@ -0,0 +1,111 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2018, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_BLOCK_DEVICE_SPISTACK_FLASH_HPP
+#define MODM_BLOCK_DEVICE_SPISTACK_FLASH_HPP
+
+#include <algorithm>
+#include <cstdlib>
+
+#include <modm/architecture/interface/block_device.hpp>
+#include <modm/processing/resumable.hpp>
+
+namespace modm
+{
+
+/**
+ * \brief	SpiStack homogenoues memory
+ *
+ * The `read()`, `erase()`,`program()` and `write()` methodes wait for
+ * the chip to finish writing to the flash.
+ *
+ * \tparam SpiBlockDevice		Base SPI block device of the homogenous stack
+ *
+ * \ingroup	modm_driver_block_device_spistack_flash
```

Comment:
```suggestion
 * \ingroup	modm_driver_block_device_spi_stack_flash
```
I think the . in the module name is translated to _ by the doc generator. Please check if this class is in the right doxypress module in the example.

Reply:
Ups yeah. This is also how its done in BdSpiFlash...